docs/ios/xcsync.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Xcsync for .NET MAUI
+description: [Xcsync is designed to provide .NET developers with improved support for editing Apple specific files]
+author: haritha-mohan, mcumming
+ms.date: [08/05/2024]
+---
+# Xcsync: improved support for Apple development via .NET
+
+Xcsync is a tool designed to provide developers using the .NET MAUI/.NET Apple SDKs have improved support for editing Apple specific files, such as storyboards and assets. The tool generates a temporary Xcode project from a .NET project, monitors both for changes, and synchronizes the changes between them.
+
+#### How to use Xcsync
+
+The tool has 2 commands right now: generate and sync. Use generate for tracking your changes from your .NET to Xcode and sync for tracking your changes from Xcode to .NET.<br />
+Xcsync can be invoked via a dotnet build target, like so:
+```dotnet build /t:xcsync-generate``` or ```dotnet build /t:xcsync-sync```
+
+You can also add the xcsync tool path to your PATH environment variable to invoke the tool directly as a more seamless dotnet command.
+
+You can customize several things regarding the tool's functionality like where the temporary Xcode project will be generated, if you would like to open the Xcode project upon being generated, how verbose you would like the logging output to be. Run ```xcsync [generate/sync] -h``` to learn more.
+
+- - - -
+
+#### Known issues
+
+* The addition of Xcsync to the workload as it is now has made it quite large. In the next release, we plan to optimize how intermediate dependencies are included and make the workload size more manageable.
```

Comment:
```suggestion
* The addition of Xcsync to the workload has increased the overall install and download size of the workloads significantly. In the next release, we plan decrease the overall installed and download workload size by 75% or more.
```

Reply:
Hmm, can we provide some real numbers? What is "quite large" and "significant"? Did the workload go from 100MB to 1GB? I think providing some real numbers would be better. Of course adding something to the workload will make it bigger, so I'm not sure if this is worth calling attention to just yet.

Now that I'm writing this, how about we keep the Known issues for the release notes, and not include that in the documentation.

Reply:
ok per both of your reviews, i'll remove the next steps and known issues for the docs and we can save it for release notes/blog post purposes.

@davidortinau i'll try to gather more precise data regarding the size increase!

Reply:
I added two screenshots of before xcsync and after xcsync to the loop document